src/app/(simulation)/(large-layout-nosticky)/fin/_components/AgirSecondaryBlock.tsx

```diff
@@ -0,0 +1,75 @@
+import Logo from '@/components/misc/Logo'
+import Trans from '@/components/translation/Trans'
+import { endClickJagisSecondBlock } from '@/constants/tracking/pages/end'
+import Button from '@/design-system/inputs/Button'
+import Badge from '@/design-system/layout/Badge'
+import { useSendSimulationToAgir } from '@/hooks/simulation/useSendSimulationToAgir'
+import { useClientTranslation } from '@/hooks/useClientTranslation'
+import { trackEvent } from '@/utils/matomo/trackEvent'
+import Image from 'next/image'
+import { useEffect } from 'react'
+
+export default function AgirSecondaryBlock() {
+  const { t } = useClientTranslation()
+
+  const { sendSimulation, data, isPending, isSuccess, isError, error } =
+    useSendSimulationToAgir()
+
+  useEffect(() => {
+    if (data?.redirectUrl && isSuccess) {
+      window.open(data.redirectUrl, '_blank')
```

Comment:
I don't think you can do this. It should be in the `click` life cycle 

https://stackoverflow.com/questions/14643040/allow-window-open-to-open-new-window-and-not-popup

Reply:
The browser decision is [apparently based on the delay between a click and the opening](https://www.mikepalmer.dev/blog/open-a-new-window-without-triggering-pop-up-blockers). Here, you click on the button before opening the page. I tested on Chrome (desktop and mobile), Safari and Firefox without issues

Reply:
Ok fair enough !